redis/server.cc

```diff
@@ -50,6 +50,23 @@ redis_server::redis_server(distributed<service::storage_proxy>& proxy, distribut
     , _memory_available(_max_request_size)
     , _auth_service(auth_service)
 {
+    namespace sm = seastar::metrics;
+
+    _metrics.add_group("redis_transport", {
```

Comment:
If this patch is about adding statistics, maybe you can move log-related stuff from the previous huge patch, to this one.
Not critical, though.

Reply:
OK.